### PR TITLE
add Genome

### PR DESCRIPTION
**Pull request by wanewang**
add https://github.com/LoganWright/Genome
and also add Gloss to README
the performance looks almost the same with Decodable

### CODE REVIEW

File: ModelObjects/Program+Gloss.swift [hunk 1]
```diff
@@ -11,6 +11,7 @@ extension Program: Decodable {
 
         title       = "Title" <~~ json ?? ""
         chanId      = "channel.ChanId" <~~ json ?? ""
+        print(chanId)
```
Comment (bwhiteley) [inline ModelObjects/Program+Gloss.swift]: This was probably unintended.

File: JSONShootout.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -110,6 +118,11 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		2098D5491DC1F5E800A15075 /* Genome.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; path = Genome.framework; sourceTree = BUILT_PRODUCTS_DIR; };
+		20A0B9561DC2584D001D8A14 /* Recording+Genome.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "Recording+Genome.swift"; sourceTree = "<group>"; };
+		20A0B9581DC25B60001D8A14 /* Program+Genome.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "Program+Genome.swift"; sourceTree = "<group>"; };
+		20A0B95A1DC25F03001D8A14 /* Genome_Test.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = Genome_Test.swift; sourceTree = "<group>"; };
+		20A0B9691DC26803001D8A14 /* GenomeFoundation.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = GenomeFoundation.framework; path = "../../../Library/Developer/Xcode/DerivedData/JSONShootout-abxsvmkqhktiehcjewcyfhuslfud/Build/Products/Debug-iphonesimulator/GenomeFoundation.framework"; sourceTree = "<group>"; };
```
Comment (bwhiteley) [inline JSONShootout.xcodeproj/project.pbxproj]: The path to derived data needs to be removed and "<group>" changed to BUILT_PRODUCTS_DIR.
See http://www.openradar.me/radar?id=6091575503355904

**Review by bwhiteley — CHANGES_REQUESTED** [review]
Note the two inline comments.

**Comment by wanewang** [comment]
I forgot to remove the test print code! I fixed it and also the framework path!

**Pull request merged**